lang/eo.yml

```diff
@@ -83,6 +83,7 @@ eo:
     NOTBLANK: 'Ne vaka'
     NOTVALUE: 'Ne valoro'
     OPTIONS: Agordoj
+    PLACEHOLDER: Lokokupilo
```

Comment:
I'm not 100%, but I think that removing old translations from non English translations files is fine because Transifex will only merge in new values, but to add the new values they'll need to be added in Transifex - @dhensby do you know more about this?